### ISSUE TITLE
Asymetric CSV encoding/decoding

### ISSUE THREAD

**Issue by luke-clifton** [begriffs/postgrest#1043]
If you want to import a `NULL` you have to use the string `NULL` in the CSV, but when downloading a CSV, `NULL` is converted to an empty string.

Is this expected/intended behaviour? I want to be able to round trip the CSVs.

**Comment by begriffs** [comment]
Yeah it's inconsistent. Not sure the best way to resolve it either, should we `COALESCE(col, 'NULL')` for each column in the csv output?

**Comment by bwbroersma** [comment]
`NULL` in CSV has no meaning, mostly `,,` is treated as a `NULL`-value, while `,"",` can be treated as empty string.
